lib/utils/index.js

```diff
@@ -174,26 +174,18 @@ class Utils {
    * @param f
    * @returns {string}
    */
-  static format(f) {
-    let i = 1;
-    let args = arguments;
-    let len = args.length;
-
+  static format(f,selector,timeMS) {
```

Comment:
can we just make this 'f' to call 'message' instead

Reply:
yeah, will make the change ✌🏻